src/category_theory/abelian/exact.lean

```diff
@@ -306,4 +307,26 @@ instance reflects_exact_sequences_of_preserves_zero_morphisms_of_faithful (F : C
 
 end functor
 
+section abelian
+
+open limits abelian
+
+universes u v
+
+variables {A B : Type u} [category.{v} A] [category.{v} B]
+variables [has_zero_object A] [has_zero_morphisms A] [has_images A] [has_equalizers A]
+variables [has_cokernels A] [abelian B]
+variables (L : A ⥤ B) [preserves_finite_limits L] [preserves_finite_colimits L]
+
+lemma exact_of_exact_functor {X Y Z : A} (f : X ⟶ Y) (g : Y ⟶ Z) (e1 : exact f g) :
```

Comment:
Can we change the name to enable dot-notation? Either `functor.map_exact` or `exact.map`?